storage/flush.go

```diff
@@ -89,8 +86,24 @@ func (m *flushManager) Flush(curr time.Time) error {
 
 	multiErr := xerrors.NewMultiError()
 	for _, ns := range namespaces {
+		// Flush first because we will only snapshot if there are no outstanding flushes
```

Comment:
talked offline: should track when we flush v snapshot in separate gauges.

Reply:
done